pom.xml

```diff
@@ -252,6 +253,7 @@
         <logback.version>1.2.3</logback.version>
         <lucene.version>7.4.0</lucene.version>
         <lux.version>1.2</lux.version>
+        <lux.saxon.version>9.5.1-3</lux.saxon.version>
```

Comment:
✏️ We might want to add a comment that this version is required for Solr if I am not mistaking ... @pklinef

Reply:
Looking through the dependency tree, the saxon dependency is coming from lux, not solr.

Reply:
... then replace `"Solr"` with `"Lux"` in my comment above 😄